op-proposer/proposer/driver.go

```diff
@@ -310,8 +310,13 @@ func proposeL2OutputTxData(abi *abi.ABI, output *eth.OutputResponse) ([]byte, er
 		new(big.Int).SetUint64(output.Status.CurrentL1.Number))
 }
 
-func (l *L2OutputSubmitter) ProposeL2OutputDGFTxData(output *eth.OutputResponse) ([]byte, error) {
-	return proposeL2OutputDGFTxData(l.dgfABI, l.Cfg.DisputeGameType, output)
+func (l *L2OutputSubmitter) ProposeL2OutputDGFTxData(output *eth.OutputResponse) ([]byte, *big.Int, error) {
+	bond, err := l.dgfContract.InitBonds(&bind.CallOpts{}, l.Cfg.DisputeGameType)
+	if err != nil {
+		return make([]byte, 0), nil, err
+	}
+	data, err := proposeL2OutputDGFTxData(l.dgfABI, l.Cfg.DisputeGameType, output)
+	return data, bond, err
```

Comment:
Believe it's an antipattern to leak data if there's an error, should probably explicitly error if there is one
```suggestion
	data, err := proposeL2OutputDGFTxData(l.dgfABI, l.Cfg.DisputeGameType, output)
	if err != nil {
	    return nil, nil, err
	}
	return data, bond, err
```